gossipd/routing.c

```diff
@@ -935,6 +935,13 @@ void handle_channel_update(struct routing_state *rstate, const u8 *update)
 		     type_to_string(trc, struct short_channel_id,
 				    &short_channel_id),
 		     flags & 0x01);
+	if (c->active != ((flags & ROUTING_FLAGS_DISABLED) == 0))
```

Comment:
This may result in a lot of output (10k lines on testnet currently), I'd rather only print this for local connections.

Reply:
OK, will fix.

Reply:
Was easier to modify existing log to print ACTIVE/DISABLED than to figure out what were local.